api/Window.json

```diff
@@ -1632,7 +1632,7 @@
               "version_added": true
             },
             "edge": {
-              "version_added": "≤18"
+              "version_added": "12"
```

Comment:
Can you double check this? I’d be surprised if devicemotion and deviceorientation events were really supported. Would that be for laptops with motion sensors then?

Reply:
I have no way to directly test this since I have no mobile Windows device, but this matches with `ondevicemotion`, and the collector agrees with its results.

Reply:
OK, let's go with that then :)